### DISCUSSION TITLE
SimpleMP to showcase

### DISCUSSION THREAD

**Discussion by lighttigerXIV** [catppuccin/catppuccin#1931] [General]
Can you add my app to the app showcase? I forgot to ask :P

https://github.com/lighttigerXIV/SimpleMP-Compose

**Comment by sgoudham** [comment]
Hi there :wave:, cool app! 

It would be great if you could pull request the repository with your app added to the list in the given format!

**Reply by lighttigerXIV** [reply]
You want me to clone catppuccin and do a PR?

**Reply by lighttigerXIV** [reply]
Ok i think i did it right
